cluster/cluster.go

```diff
@@ -118,6 +122,9 @@ func (c *Cluster) setClusterDefaults() {
 			// This is a temporary modification
 			c.Nodes[i].HostnameOverride = c.Nodes[i].Address
 		}
+		if len(host.SSHKeyPath) == 0 {
```

Comment:
should we default to cluster only when both sshkeypath and sshkey are empty? As user can pass either of those

Reply:
ssh key will have the priority over either of options, so i doesn't matter if the user only passed ssh key, because it will be used instead of any path